components/ScoreTable/index.tsx

```diff
@@ -22,20 +22,36 @@ interface Props {
   data: Array<Player | Goalie>;
   columnData: Array<ColumnData>;
   pagination?: boolean;
+  teamPage?: boolean;
   // isLoading: boolean;
 }
 
 function ScoreTable({
   data: players,
   pagination = false,
   columnData,
+  teamPage = false,
 }: // isLoading
 Props): JSX.Element {
   // ! add loading state
   const data = useMemo(() => players, [players]);
 
   // ! add loading state
-  const columns = useMemo(() => columnData, []);
+  const columns = useMemo(() => {
```

Comment:
this is the logic that does all the magic.  Essentially, if a teamPage is used, loop through the columns of the first section.  If the header equals Team, remove it.